### ISSUE TITLE
Allow specification of experiment runner version using release tag instead of commit SHA

### ISSUE THREAD

**Issue by kimberscott** [lookit/lookit-api#619]
# TL;DR
Allow researchers to specify an experiment runner version using a release tag instead of a commit SHA

# Narrative
As a researcher, I would like to keep my experiment runner up to date and understand if there are new features or bugfixes that I would benefit from, but it's hard to tell what I'm using compared with the latest release. This makes it harder to know e.g. which release notes to look through.

As a maintainer of the ember-lookit-frameplayer codebase, I would like to be able to see more clearly which versions are in active use.

# Acceptance Criteria

- [ ] Researcher can specify release tag for experiment runner version
- [ ] Release tag is stored and visible as metadata, either replacing or in addition to commit SHA
- [ ] Data is fetched and displayed about commit corresponding to release tag, as with commit SHA currently
- [ ] Researcher can still specify commit SHA instead if desired
